thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js

```diff
@@ -136,6 +136,8 @@ export default function(server) {
    * should return mocked data
    */
   server.get('/rootcause/metric/aggregate', () => {});
+  server.get('/rootcause/metric/aggregate/cache', () => {});
+  server.get('/rootcause/metric/aggregate/batch', () => {});
   server.get('/rootcause/metric/breakdown', () => {});
   server.get('rootcause/metric/aggregate/batch', () => {});
```

Comment:
looks like a duplicate. maybe rebase the PR to master?

Reply:
sure